explorer/src/pages/TransactionDetailsPage.tsx

```diff
@@ -231,7 +283,7 @@ function AccountsCard({ signature }: Props) {
 
   if (!status?.data?.info) {
     return null;
-  } else if (!details) {
+  } else if (!details || status.data.info.confirmations !== "max") {
```

Comment:
What if we showed the loading card while auto refresh is active? The error card shows a retry button which a user shouldn't need to bother with when auto refreshing

Reply:
That would make sense.

Reply:
I think this might involve placing autoRefreshInProcess back into the context? Will discuss later.

Reply:
So basically, there are three overall states: transaction not found, transaction hasn't reached max (auto refresh active), transaction has max confirmations.

The "details are not available until the transaction reaches MAX confirmations" error card would be eliminated.